internal/environment/guac.go

```diff
@@ -102,9 +102,9 @@ func (guac *Guacamole) create(ctx context.Context, eventTag string) error {
 	guacdAlias := uuid.New().String()
 	dbAlias := uuid.New().String()
 	containers["web"] = docker.NewContainer(docker.ContainerConfig{
-		Image: "registry.gitlab.com/haaukins/core-utils/guacamole",
+		Image: "guacamole/guacamole:1.4.0",
 		EnvVars: map[string]string{
-			"MYSQL_DATABASE": "guacamole_db",
+			"MYSQL_DATABASE": "guacamole_db?useSSL=false",
```

Comment:
Is this just for testing? I know it is host only but isn't it best practice to always use ssl if possible?

Reply:
I found no evidence in our previously created guac containers that we are using SSL. But for some reason the new version of Guac requires us to explicitly set the useSSL parameter in order to disable it...

Reply:
And the container is not even exposed to localhost. There is no real sensitive information in the guac databases anyways. So the risk and impact of a compromise is very very low

Reply:
The reason is new Database standards :)

Reply:
Fair, It is just nice to have these considerations written in the git history :)